app/src/Api.tsx

```diff
@@ -195,4 +194,65 @@ export class FirebaseService {
    *    - start with entries: 1, or increment the number of entries
    * 
    */
-}
\ No newline at end of file
+}
+
+export type Task = {
+  title: string;
+  description: string;
+  borough: string;
+  index: number;
+}
+
+export type Prize = {
+  prizeTitle: string;
+  prizeSubtitle: string;
+  description: string;
+  dollarValue: string;
+  ticketValue: number;
+  productLink: string;
+  imageUrl: string;
+}
```

Comment:
For both of these, can you also extract and save the `id`? it's not in the fields level, but rather on the object level, looks like `rec2DI12lrbNx5Bgm`. We'll need it for saving the user data on which tasks and raffles they've interacted with